mods/ts/mod.yaml

```diff
@@ -20,7 +20,7 @@ MapFolders:
 
 Packages:
 # Tiberian Sun
-	~scores.mix:CRC32
+	~SCORES.MIX:CRC32
```

Comment:
We have lower-case package lists in all other C&C mod so let's keep it that way for consistency.